test/tobs/test_list.py

```diff
@@ -148,10 +148,10 @@ def test_list_cached_mappings(capsys):
     # Reducing the large output since otherwise the output is hundreds of lines.
     ListScript("crds.list --cached-mappings --full-path")()
     out, _ = capsys.readouterr()
-    out_to_check1 = '/grp/crds/cache/mappings/jwst/jwst.pmap'
-    out_to_check2 = '/grp/crds/cache/mappings/jwst/jwst_miri_photom_0027.rmap'
-    out_to_check3 = '/grp/crds/cache/mappings/jwst/jwst_niriss_abvegaoffset_0001.rmap'
-    out_to_check4 = '/grp/crds/cache/mappings/jwst/jwst_nirspec_superbias_0022.rmap'
+    out_to_check1 = '/mappings/jwst/jwst.pmap'
+    out_to_check2 = '/mappings/jwst/jwst_miri_photom_0027.rmap'
+    out_to_check3 = '/mappings/jwst/jwst_niriss_abvegaoffset_0001.rmap'
+    out_to_check4 = '/mappings/jwst/jwst_nirspec_superbias_0022.rmap'
```

Comment:
This is a case where you would need to use the "jwst_shared_cache_state" fixture. For example:

```python
def test_list_cached_mappings(jwst_shared_cache_state, capsys):
    # Reducing the large output since otherwise the output is hundreds of lines.
    cache = jwst_shared_cache_state.cache
    ListScript("crds.list --cached-mappings --full-path")()
    out, _ = capsys.readouterr()
    jwst_shared_cache_state.cleanup()
    out_to_check1 = f'{cache}/mappings/jwst/jwst.pmap'
    out_to_check2 = f'{cache}/mappings/jwst/jwst_miri_photom_0027.rmap'
    out_to_check3 = f'{cache}/mappings/jwst/jwst_niriss_abvegaoffset_0001.rmap'
    out_to_check4 = f'{cache}/mappings/jwst/jwst_nirspec_superbias_0022.rmap'
    assert out_to_check1 in out, out_to_check2 in out
    assert out_to_check3 in out, out_to_check4 in out
```

Reply:
Done.